src/ucp/wireup/wireup.c

```diff
@@ -1433,9 +1432,16 @@ ucp_wireup_check_config_intersect(ucp_ep_h ep, ucp_ep_config_key_t *new_key,
         ucs_assert(ucp_ep_get_lane(ep, lane) == NULL);
     }
 
-    for (lane = 0; lane < UCP_MAX_LANES; ++lane) {
+    status = ucp_ep_realloc_lanes(ep, new_key->num_lanes);
+    if (status != UCS_OK) {
+        return status;
```

Comment:
we should clean up new_uct_eps content

Reply:
it's a local var (on stack). why need to clean?

Reply:
what about the eps that it contains?